css/main.css

```diff
@@ -51,6 +51,113 @@ header {
   color: white;
 }
 
+header {
+  display: flex;
+}
+
+.navbar {
+  /* display: flex; */
+  justify-content: center;
+  align-items: center;
+  background-color: #000;
+  color: #fff;
+  border-bottom: 1px solid rgba(255,255,255,0.2);
+  width: 100%;
+  height: 80px;
+  line-height: 80px;
+  font-size: 18px;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.navbar a.navbar-brand {
+  line-height: inherit;
+  /* padding: 0 20px; */
+  font-size: 22px;
+  text-transform: uppercase;
+  font-weight: 400;
+  letter-spacing: 2px;
+}
+
+.navbar h1 {
+  padding-left: 20px;
+}
+
+.navbar ul {
+  width: 100%;
+  /* float: right; */
+  list-style: none;
+  /* height: inherit; */
+  line-height: inherit;
+  padding: 0;
+  background-color: rgba(0,0,0,0.9);
+  /* height: auto; */
+  height: 60vh;
+  max-height: 0;
+  overflow: hidden;
+  transition: all ease-in-out 0.3s;
+}
+.navbar ul li {
+  display: inline-block;
+}
+.navbar ul li a {
+  display: block;
+  text-align: center;
+  min-width: 120px;
+  width: 100%;
+  padding: 0 30px;
+}
+.navbar ul li a:hover {
+  background-color: rgba(255,255,255,0.15);
+}
+
+.navbar .navbar-toggler {
+  display: none;
+}
+
+
+
+.navbar ul.open {
+  max-height: 100vh;
+}
+
+.navbar ul li {
+  width: 100%;
+  /* border-bottom: 1px solid rgba(255,255,255,0.3); */
+}
+/* .navbar ul li a {
+  padding: 0;
+} */
+.navbar .navbar-toggler {
+  display: block;
+  position: absolute;
+  /* height: 40px; */
+  top: 20px;
+  right: 20px;
+  background-color: transparent;
+  color: #fff;
+  border: 3px solid #fff;
+  /* border: none; */
+  /* border-radius: 4px; */
+  outline: none;
+  padding: 8px;
+  cursor: pointer;
+}
```

Comment:
- [ ] _[Required]_ As I stated earlier, kindly remove all commented out styles from your CSS file. You can use Git log if you need to go back and see certain styles you added before.